.github/stale.yml

```diff
@@ -0,0 +1,17 @@
+# Number of days of inactivity before an issue becomes stale
+daysUntilStale: 365
+# Number of days of inactivity before a stale issue is closed
+daysUntilClose: 7
```

Comment:
I think since we have such a high amount of days for stale issues right now, we could give folks a little more time to alert us:
```suggestion
daysUntilClose: 14
```

Reply:
Yeah that makes sense